packages/titanium-search-input/src/titanium-search-input.ts

```diff
@@ -130,13 +130,9 @@ export class TitaniumSearchInput extends LitElement {
     }
 
     :host([collapsed]:not([prevent-collapse])) input-container input {
-      display: none;
-    }
-
-    :host([collapsed]:not([prevent-collapse])) titanium-svg-button[search] {
-      position: relative;
-      top: 0;
-      left: 0;
+      opacity: 0;
+      pointer-events: none;
+      z-index: 1;
```

Comment:
why z-index?  could you rearrange  titanium-svg-button's instead?

Reply:
You are right.  I don't need the z-index.  The pointer-events none makes this not needed.  I added that after adding the z-index and should have removed it.  Thank you, nice catch!

Reply:
This is now removed.